doc/Usage.adoc

```diff
@@ -0,0 +1,272 @@
+= Usage
+
+:toc:
+:toc-placement!:
+
+The following sections explain how you can use the Landrush DNS server and customize it as per your requirements, once you have link:README.adoc[installed] it.
+
+'''
+toc::[]
+'''
+
```

Comment:
We have links to headings like "Dynamic entries", "Unmatched Queries", etc, but no list of actual options. Each section uses/introduces options, eg `config.landrush.host_interface_excludes` or `config.landrush.host`. These list of Landrush config options it distributed across the whole document with no easy means of getting an overview what exists. I would find such a table very useful, whether a table row entry then links to a specific section or not is a different story. A config options index of forms. But if you don't like it, we leave things as they are.